etc/bash-utils.sh

```diff
@@ -436,6 +443,22 @@ build-cuspatial-python() {
 
 export -f build-cuspatial-python;
 
+build-blazingsql() {
+    if [ ! -d "$BLAZINGSQL_HOME/thirdparty/cudf" ]; then 
+        # link to host cudf? this causes issues with running 
+        # BlazingSQLs build script if you're running on a fork
+        #ln -s $CUDF_HOME $BLAZINGSQL_HOME/thirdparty/cudf
+        
+        ln -s $BLAZINGSQL_HOME $CONDA_PREFIX/blazingsql
+    fi
```

Comment:
FWIW, I have this PR open for blazing which might help when it gets merged: https://github.com/BlazingDB/blazingsql/pull/849

Reply:
Thanks @rlratzel :pray:. I mentioned to @millerhooks on slack that it may be a good idea if we build BlazingSQL's CMakeLists.txt directly, since we may need to override more library or include paths.

Reply:
Thanks @rlratzel. That is spot on and looks like it's been merged!